library/ssl_tls13_generic.c

```diff
@@ -1855,8 +1856,10 @@ static int ssl_certificate_verify_write( mbedtls_ssl_context* ssl,
     /* Verify whether we can use signature algorithm */
     ssl->handshake->signature_scheme_client = SIGNATURE_NONE;
 
+#if defined(MBEDTLS_SSL_SRV_C)
```

Comment:
This change leads to a compile-time error because the `sig_alg` variable is now set but not used.